include/oneapi/dpl/internal/random_impl/bernoulli_distribution.h

```diff
@@ -208,4 +208,4 @@ class bernoulli_distribution
 } // namespace dpl
 } // namespace oneapi
 
-#endif // #ifndf _ONEDPL_BERNOULLI_DISTRIBUTION
\ No newline at end of file
+#endif // #ifndf _ONEDPL_BERNOULLI_DISTRIBUTION_H

```

Comment:
#ifndf should probably be removed for consistency, there are a number of these.

./include/oneapi/dpl/internal/random_impl/bernoulli_distribution.h:211:#endif // #ifndf _ONEDPL_BERNOULLI_DISTRIBUTION_H
./include/oneapi/dpl/internal/random_impl/cauchy_distribution.h:221:#endif // #ifndf _ONEDPL_CAUCHY_DISTRIBUTION_H
./include/oneapi/dpl/internal/random_impl/exponential_distribution.h:213:#endif // #ifndf _ONEDPL_EXPONENTIAL_DISTRIBUTION_H
./include/oneapi/dpl/internal/random_impl/extreme_value_distribution.h:245:#endif // #ifndf _ONEDPL_EXTREME_VALUE_DISTRIBUTION_H
./include/oneapi/dpl/internal/random_impl/geometric_distribution.h:210:#endif // #ifndf _ONEDPL_GEOMETRIC_DISTRIBUTION_H
./include/oneapi/dpl/internal/random_impl/lognormal_distribution.h:236:#endif // #ifndf _ONEDPL_LOGNORMAL_DISTRIBUTION_H
./include/oneapi/dpl/internal/random_impl/normal_distribution.h:424:#endif // #ifndf _ONEDPL_NORMAL_DISTRIBUTION_H
./include/oneapi/dpl/internal/random_impl/uniform_int_distribution.h:221:#endif // #ifndf _ONEDPL_UNIFORM_INT_DISTRIBUTION_H
./include/oneapi/dpl/internal/random_impl/uniform_real_distribution.h:375:#endif // #ifndf _ONEDPL_UNIFORM_REAL_DISTRIBUTION_H
./include/oneapi/dpl/internal/random_impl/weibull_distribution.h:222:#endif // #ifndf _ONEDPL_WEIBULL_DISTRIBUTION_H

Reply:
Fixed